panel/src/components/Forms/Input/NumberInput.vue

```diff
@@ -84,6 +85,10 @@ export default {
         return 0;
       }
 
+      if (step.toString().indexOf('e') !== -1) {
+        return parseInt(step.toFixed(16).split(".")[1].split("").reverse().join("")).toString().length;
+      }
```

Comment:
Steps of method that i found:

1. Check float: 1e-6
2. Fix with 16 decimals as `0.0000010000000000`
3. Get number decimals as `0000010000000000`
4. Reverse the number: `0000000000100000`
5. ParseInt as `100000`
6. Get length as `6`

Wohooo 😄